### PR TITLE
adjust camera for VR/2D mode change

### PR DESCRIPTION
**Pull request by caseyyee**
vr:
image

2d:
image

### CODE REVIEW

File: examples/animation-aframe-logo/index.html [hunk 1]
```diff
@@ -199,4 +199,32 @@
 
     </a-scene>
   </body>
+  <script>
+  // Change camera setting entering/exiting VR mode.
```
Comment (cvan) [inline examples/animation-aframe-logo/index.html]: indent

**Comment by cvan** [comment]
do you still want to merge this?

**Comment by caseyyee** [comment]
Depends on what you guys want to do with it.     If you are OK not having a VR view or have a different way about this, then by all means, blow it away.

**Comment by ngokevin** [comment]
This looks sweet. The scene has an API to set active camera now, let's look into that for next release.

**Comment by caseyyee** [comment]
Closing for now until we have a better solution

**Pull request closed without merging**